docs/CODEOWNERS

```diff
@@ -1,21 +1,21 @@
-# Subsystems have specific owners
-
-
-CMake/                                      @robertmaynard
-data/                                       @dpugmire
-docs/                                       @robertmaynard @kmorel
-examples/                                   @kmorel
-Utilities/                                  @kmorel
-viskores/                                       @robertmaynard @kmorel
+.github/                                        @vicentebolea
+.gitlab-ci.yml                                  @vicentebolea
+.gitlab/                                        @vicentebolea
+CMake/                                          @vicentebolea
+data/                                           @dpugmire
+docs/                                           @robertmaynard @kmorel
+examples/                                       @kmorel
+Utilities/                                      @kmorel @vicentebolea
 viskores/interop/                               @robertmaynard
```

Comment:
I'm tempted to change this to @jeffamstutz. He probably knows as much about it as anyone.